commands/debug.js

```diff
@@ -62,43 +62,65 @@ if(args[0].toLowerCase()==="quest"){
   message.channel.send("This creature can't do quests.");
   return;
 }
+
+var messagePing = message.mentions.members.first();
+var targetID = messagePing ? message.guild.id.concat(messagePing.id) : userid;
+charid = client.userMap.get(targetID,"possess");
+sburbid = client.charcall.allData(client,targetID,charid,"owner");
+
 if(args[0].toLowerCase()==="capgrist"){
-  var gristCheck = client.charcall.allData(client,userid,charid,"grist");
+  var gristCheck = client.charcall.allData(client,targetID,charid,"grist");
+  if(gristCheck == "NONE")
+  {
+    message.channel.send(`That person isn't currently possessing anything that has grist.`);
+    return;
+  }
+
+  if(sburbid == "NONE")
+  {
+    message.channel.send(`That one simply cannot be helped right now.`);
+    return;
+  }
+
   const gristTypes = ["build","uranium","amethyst","garnet","iron","marble","chalk","shale","cobalt","ruby","caulk","tar","amber","artifact","zillium","diamond"];
-  let rung = client.sburbMap.get(sburbid,"rung");
-  
   let max = 0;
-  if (client.sburbMap.get(sburbid,`godtier`)) {
-	if(args[1].toLowerCase()!="confirm")
-	{
-        message.channel.send(`I can only help godtiers by setting their invalid grist levels to 0. If you're sure, add a 'confirm' to that.`);
-		return;
-	}
+  if(sburbid && !client.sburbMap.get(sburbid,`godtier`))
```

Comment:
Here's one of the fun things about the data rework, things can be built with the idea that possibly, an NPC might have godtier, or a grist cap, even though that's usually stored in Sburbdata. to account for this, instead of checking sburbMap, we can use allData to see if there's a godtier boolean defined anywhere on any relevant sheets. sburbid pretty much never needs to be directly referenced.